Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -363,6 +366,8 @@ extension TopBannerViewController: NavigationComponent {
     
     public func navigationService(_ service: NavigationService, didPassVisualInstructionPoint instruction: VisualInstructionBanner, routeProgress: RouteProgress) {
         instructionsBannerView.update(for: instruction)
+        currentInstruction = instruction
+        lanesView.isTopBannerDisplayingSteps = isDisplayingSteps
         lanesView.update(for: instruction)
         nextBannerView.navigationService(service, didPassVisualInstructionPoint: instruction, routeProgress: routeProgress)
         junctionView.update(for: instruction, service: service)
```

Comment:
Looks like all of these views are affected. I think that there is even no need to update them when `isDisplayingSteps` is set to `true`. So we can add something like:
```suggestion
        if isDisplayingSteps {
            return
        }
        
        lanesView.update(for: instruction)
        nextBannerView.navigationService(service, didPassVisualInstructionPoint: instruction, routeProgress: routeProgress)
        junctionView.update(for: instruction, service: service)
```